default/AI/ResearchAI.py

```diff
@@ -160,11 +173,13 @@ def get_max_stealth_species():
     result = (stealth_species, stealth)
     return result
 
+
 def get_initial_research_target():
     # TODO: consider cases where may want lesser target
     return AIDependencies.ART_MINDS
 
-def get_ship_tech_usefulness(tech, ship_designer):
+
+def get_ship_tech_usefulness(tech, ship_designer, tech_name=""):
```

Comment:
get_ship_tech_usefulness is not used directly as a priority func, and doesn't need this extra tech_name parameter, although if you want to rename the first parameter 'tech' to 'tech_name' (which is what it really is, not the tech itself), that would probably be good.